ci-operator/jobs/openshift/cloud-provider-openstack/openshift-cloud-provider-openstack-master-presubmits.yaml

```diff
@@ -11,6 +11,7 @@ presubmits:
       ci-operator.openshift.io/prowgen-controlled: "true"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-cloud-provider-openstack-master-e2e-openstack
+    optional: true
```

Comment:
Didn't you mean to make `e2e-openstack-csi-cinder` optional instead of `e2e-openstack`?

Reply:
yes thanks